File: 프로그래머스/2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = []
    count = 0
    
    for i, price in enumerate(prices):
        for j in range(i + 1, len(prices)):
            if price > prices[j]:
                count += 1
                break
            else:
                count += 1
        answer.append(count)
        count = 0
    
    return answer

File: 프로그래머스/2/42584. 주식가격/주식가격.js
function solution(prices) {
    var answer = [];
    
    for(let i = 0; i < prices.length; i++){
        let currentPrice = prices[i];
        let cnt = 0;
        
        for(let j = i + 1; j < prices.length; j++){
            cnt++;
            if(currentPrice > prices[j]) break; 
        }
        answer.push(cnt);
    }
    return answer;
}